javascript/javascript2/week1/shortword.js

```diff
@@ -0,0 +1,16 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const shortWord = getShortestWord(danishWords);
+
+function getShortestWord(words) {
+  function getShortestWord() {
+    let newWord = danishWords[0];
+    for (let i = 1; i < danishWords.length; i++) {
+      if (danishWords[i].length <= newWord.length) {
+        newWord = danishWords[i];
+      }
+    }
+    return newWord;
+  }
+  document.getElementById("ShortestWord").innerHTML =
+    "Shortest word in array is : " + shortWord;
+}
```

Comment:
Great, maybe a little imporvment can be made by making this function little more generic. This works for the static array of words given, i suggest you to make the function not to be statically working with only this static array, instead it could take any carry of words and compare them and return the result. Nice Job though!